pkg/remote/remote_test.go

```diff
@@ -106,22 +105,20 @@ func TestWithDetection_sshConfig_Hostname(t *testing.T) {
 	shutdown, host, port := testServer(t, false)
 	defer shutdown()
 
-	sshConfigFile, _ := homedir.Expand("~/.ssh/config")
+	tmpFile, err := os.CreateTemp(t.TempDir(), "")
+	require.NoError(t, err)
+
+	defer tmpFile.Close()
+
+	sshConfigFile := tmpFile.Name()
```

Comment:
It's being used once in the func so don't need to create a new variable for that, simply use `tmpFile.Name()` instead.